rtrlib/lib/utils.c

```diff
@@ -13,14 +13,14 @@
 #include <stdint.h>
 #include <time.h>
 
-#ifdef __MACH__
+#if defined(__MACH__) && defined(__APPLE__)
```

Comment:
maybe this should be a different commit? i don't see the relation to vectors here.